plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/internal/core/WatchManager.java

```diff
@@ -41,7 +41,7 @@
 	private static final int[] FIBONACCI = new int[] { 0, 1, 1, 2, 3, 5, 8, 13, 21 };
 	private static final long BACKOFF_MILLIS = 5000;
 	private static final long BACKOFF_RESET = FIBONACCI[FIBONACCI.length - 1] * BACKOFF_MILLIS * 2;
-	private static final String [] KINDS = new String[] {
```

Comment:
Made public so we could reuse in the DeploymentMapper and didnt need to redefine.  I figured the mapper is interested, probably, the same kinds as the watch is generating notifications.